notebooks-html/bias_risk.html

```diff
@@ -77,76 +77,83 @@
 <div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
 <h2 id="Risk-and-Cost-Minimization">Risk and Cost Minimization<a class="anchor-link" href="#Risk-and-Cost-Minimization">&#182;</a></h2><p>In order to make predictions using data, we define a model, select a cost function, and fit the model's parameters by minimizing the cost. For example, to conduct least squares linear regression, we select the model:</p>
-$$
+<p>$$
 \begin{aligned}
 f_\hat{\theta} (x) &amp;= \hat{\theta} \cdot x
 \end{aligned}
-$$<p>And the cost function:</p>
-$$
+$$</p>
+<p>And the cost function:</p>
+<p>$$
 \begin{aligned}
 L(\hat{\theta}, X, y)
 &amp;= \frac{1}{n} \sum_{i}(y_i - f_\hat{\theta} (X_i))^2\\
 \end{aligned}
-$$<p>As before, we use $ \hat{\theta} $ as our vector of model parameters, $ x $ as a vector containing a row of a data matrix $ X $, and $ y $ as our vector of observed values to predict.</p></div></div></div>
+$$</p>
+<p>As before, we use $ \hat{\theta} $ as our vector of model parameters, $ x $ as a vector containing a row of a data matrix $ X $, and $ y $ as our vector of observed values to predict, where $ X_i $ is the ith row of $ X $ and $ y_i $ is the ith entry of y.</p></div></div></div>
 <div class="cell border-box-sizing text_cell rendered"><div class="inner_cell">
 <div class="text_cell_render border-box-sizing rendered_html">
-<p>Observe that our cost function is the average of the loss function values for each row of our data. If we define the loss function:</p>
-$$
+<p>Observe that our cost function is the average of the loss function values for each row of our data. If we define the loss function (squared loss):</p>
+<p>$$
 \begin{aligned}
-\ell(y, f_\hat{\theta} (x))
+\ell(y_i, f_\hat{\theta} (x))
 &amp;= (y_i - f_\hat{\theta} (x))^2
 \end{aligned}
-$$<p>Then we may rewrite our cost function more simply:</p>
-$$
+$$</p>
+<p>Then we may rewrite our cost function more simply:</p>
+<p>$$
 \begin{aligned}
-L(\hat{\theta}, X, y)
+L(\hat{\theta}, X, y_i)
```

Comment:
I agree with your loss function change above, but the cost function definition should take in `y` as a parameter, not `y_i`. Notice the summation over all components of `y`—this is why we need the entire `y` vector.